### PR TITLE
MyList, GraphQL, Travis CI

### PR DESCRIPTION
**Pull request by LoicVerrall**
MyList screen has been added with a primitive FlatList (table) to display favourite unis.

GraphQL request to get uni info by `pubukprn` has been added.

Travis script updated to install and run `standard`.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ language: node_js
 install:
 - echo "install nothing!"
 script:
-- echo "no tests!"
+- echo "Checking JavaScript is 'Standard'"
+- npm install -g standard
```
Comment (danarth) [inline .travis.yml]: Installing packages should not happen in the testing phase. Also, you will not have permission to install a package globally using `-g` on a Travis build. You should do the following instead:

1. On your local machine, `cd` to the `mobile` directory
2. Run `npm install --save-dev standard
3. In the `package.json`, prepend the `"test"` script with `standard && ...`
4. Commit all changes to `package.json`
5. In the `.travis.yml` file, change the `script` part to this:
```yml
script:
- npm test
```
6. Change the `install` section to:
```yml
install:
- npm install
```

**Review by danarth — DISMISSED** [review]
Thanks for all of your work on this! Just a couple of changes to .travis.yml are needed to get this ready.

File: .travis.yml (hunk 1 above)
Comment (LoicVerrall) [reply .travis.yml]: Thanks for the review — I've implemented these changes in the most recent commit.

Success! Travis fails if `standard` reports errors (see screenshot). I will fix those code style issues now - don't merge yet.

screen shot 2018-04-24 at 13 27 28

**Comment by LoicVerrall** [comment]
Code now conforms to StandardJS and this is correctly checked automatically when a PR is submitted. The Travis build is still failing because there needs to be at least one unit test.

**Review by Tang-1996 — APPROVED** [review]

**Pull request merged**